### ISSUE TITLE
Spring for GraphQL support/ sample

### ISSUE THREAD

**Issue by deki** [aws/serverless-java-container#715]
At re:Invent 2023 someone asked if we are supporting Spring for GraphQL https://docs.spring.io/spring-graphql/reference/. Goal of this issue is to check that and ideally add a sample for it.

**Comment by mbfreder** [comment]
Thanks for raising this. I will check if it's already supported.

**Comment by fredbonin** [comment]
Hi,
I can confirm that GraphQL works. 

I've used the code here to test it : 
https://spring.io/guides/gs/graphql-server/ and it just worked (I added a StreamLambdaHandle as documented here https://github.com/aws/serverless-java-container/wiki/Quick-start---Spring-Boot3).

However, I had issues when integrating with JPA. It was necessary to add the config below:

```
spring:
  jpa:
    open-in-view: false
```

If it was not added, the handler was going in an infinite loop (waiting for a latch). This was not necessary for REST API (in any case, it is good practice to disable this for all spring applications)
